### ISSUE TITLE
Treatment of feedstock time

### ISSUE THREAD

**Issue by widdowquinn** [nickp60/Nolan_et_al_2018#5]
I don't think it's valid to consider the time of introduction of feedstock to be "-1 days", as currently described in `BHM_all_results.html`:

> Due to the design of the experiment, we are going to try displaying the feedstock values at day -1, rather than day 0, as this represents our input material. Ultimately, the useful comparison is the readings of the original feedstock to the digestate that comes out at the end of the reactor. In reality, the day 0 readings for the feedstock and the day 0 reading for the reactors (after the feedstock was added )took place on the same day.

We have limited choices: 

- consider the feedstock *or* the day 0 measurement to be the "valid" day 0 measurement (this would discard data, which is not desirable)
- consider both feedstock *and* day 0 measurements to be "valid" day 0 measurements (this would combine measurements separated by an unknown amount of time, and presumably when the modelled processes are active and may be at their fastest rate; this could be problematic)
- convert times to hours, and treat feedstock as hour 0, and "day 0" as whatever hour the reading was taken (we may not have this information, and the 'hour' of other measurements could be +/- 24h as a result)
- something else I'm not thinking of…?

**Comment by nickp60** [comment]
This was solely for plotting purposes.  We decided that because the feedstock is the same for all conditions but was not measured over time, it does not represent a repeated measure and cannot be connected with the subsequent time points.  The feedstock will just serve in comparison between what goes in to the reactors and what comes out at the end.  This is why we dropped the feedstock values just after, prior to modelling:

```
spread_data <- tall %>%
  filter(condition != "FS") %>%
  spread(measure, value) %>%
  as.data.frame()
```
